docs/database.md

```diff
@@ -40,7 +40,7 @@ cd backend
 DATABASE_CONNECTION_STRING="postgresql://postgres:203d805f4b62c0a1b2f1f6b82d4583dfe563ec1619b83ce22ee414e8376a25e7@localhost:6545/postgres" PYTHONPATH=src alembic revision --autogenerate -m "Modify the database"
 
 # now format the migration
-isort . && black .
+autoflake -r -i --remove-all-unused-imports . && isort . && black .
```

Comment:
`.` -> `src`?

Reply:
This should be `autoflake -r -i --remove-all-unused-imports src`, right?

Reply:
Yup. My B. Thought I'd done it